testdata/fixtures/show_arp.json

```diff
@@ -0,0 +1,39 @@
+{
+  "jsonrpc": "2.0",
+  "id": "1",
+  "result": [
+    {},
+    {
+      "dynamicEntries": 4,
+      "ipV4Neighbors": [
+        {
+          "hwAddress": "444c.a8aa.bbcc",
+          "address": "10.10.10.10",
+          "interface": "Ethernet1/1",
+          "age": 0
```

Comment:
I'm unsure if this is actually an integer or maybe a float64. When running it on devices I just see Age set as `0` always.